t/kubernetes/discovery/kubernetes2.t

```diff
@@ -99,6 +99,21 @@ _EOC_
     $block->set_value("main_config", $main_config);
 
     my $config = $block->config // <<_EOC_;
+        location /informer {
```

Comment:
this test case doesn't make sense. It is difficult to understand what problem this PR fixes (although from the link that you shared, it appears that the url is incorrect).. so in theory... namespace based queries should fail (given the path is incorrect)...

can you show in a test case that namespace related queries pass now?

Reply:
Thanks for your review. It's difficult to verify in e2e case. Is there an example of unittest case?

Reply:
I'm not familar with this plugin, could you explain why the url you changed has anything to do with the document you are referencing?
We can ignore the test cases If they are too difficult

Reply:
Just fix Informer_factory.path splicing problem